### DISCUSSION TITLE
Cypress 9.0.0 not working on Windows 11 using scoop

### DISCUSSION THREAD

**Discussion by mrbusche** [cypress-io/cypress#18903] [General]
I have a Windows 11 PC and used scoop to install node. Using cypress 9.0.0 I receive the following error.

`The following error was thrown by a plugin. We stopped running your tests because a plugin crashed.`

Using version 8.7.0, everything works fine. Running `node -v` returns `v17.1.0`. 

Looking at the breaking changes for `v9.0.0`

- [x] nodeVersion defaults to system; `node -v` returns `v17.1.0`
- [x] windows 32 bit is not support; PC is 64 bit
- [x] Invalid `Cypress.config` will throw an error; Using default configuration
- [x] `cy.contains`; tests haven't kicked off yet
- [x] `Cypress.Commands.add()`; not using that
- [x] Custom command implementations; using default install
- [x] bundle Node.js version upgraded; Tests run fine on node `v.17.1.0`

Cypress runs fine with `v9.0.0` using [Github actions](https://github.com/mrbusche/cottage-checker/actions/runs/1456446730) with [Ubuntu 20.04](https://github.com/mrbusche/cottage-checker/blob/main/.github/workflows/cypress.yml)

Happy to provide more debug details if anyone has any idea what could be wrong.

My repository is https://github.com/mrbusche/cottage-checker

**Comment by singh-sukhpreet** [comment]
I also have the similar error 

**Title:** Error running plugin

**Message:** The following error was thrown by a plugin. We stopped running your tests because a plugin crashed. Please check your plugins file (`/Users/sukhpreetsingh/learning/cypress/cypress/plugins/index.js`)

**Stack trace:**
```
Error: The following error was thrown by a plugin. We stopped running your tests because a plugin crashed. Please check your plugins file (`/Users/sukhpreetsingh/learning/cypress/cypress/plugins/index.js`)
    at Object.get (/Users/sukhpreetsingh/Library/Caches/Cypress/9.1.0/Cypress.app/Contents/Resources/app/packages/server/lib/errors.js:1043:15)
    at EventEmitter.handleError (/Users/sukhpreetsingh/Library/Caches/Cypress/9.1.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/index.js:189:20)
    at EventEmitter.emit (node:events:394:28)
    at ChildProcess.<anonymous> (/Users/sukhpreetsingh/Library/Caches/Cypress/9.1.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/util.js:19:22)
    at ChildProcess.emit (node:events:394:28)
    at emit (node:internal/child_process:920:12)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)

```

OS - MacOS Big Sur Version 116
Node version - v17.0.1
cypress version - ^9.1.0 (install via npm - npm install cypress --save-dev)

**Comment by singh-sukhpreet** [comment]
Hi @mrbusche, I solved this problem by using the different node version. try the following version

nvm use 14.18.1 - it works for me

**Comment by mjhenkes** [comment]
Related issue: https://github.com/cypress-io/cypress/issues/18914

**Comment by mrbusche** [comment]
Closing because https://github.com/cypress-io/cypress/issues/18914 is similar
